processor/src/fast/mod.rs

```diff
@@ -963,31 +1175,29 @@ impl FastProcessor {
         Ok(())
     }
 
-    /// Analogous to [`Process::resolve_external_node`](crate::Process::resolve_external_node), but
-    /// for asynchronous execution.
-    async fn resolve_external_node(
+    // HELPERS
+    // ----------------------------------------------------------------------------------------------
+
+    async fn load_mast_forest<E>(
```

Comment:
Below we can add the `err_ctx` to the `advice_error`

Reply:
Good catch - fixed